test/helpers/get-deployed-contracts-for-test.ts

```diff
@@ -44,10 +44,14 @@ async function getDeployedContractsForTest(): Promise<AllDeployedContracts> {
     deployedContracts = await getAllContracts();
   } else {
     deployedContracts = await deployContractsForTest();
+
+    // Put proposals that have already been executed on mainnet here.
```

Comment:
Any suggestions on how to structure this comment? Want to make it obvious to people where to place proposals that have already passed (that will only be executed on Hardhat) vs upcoming proposals (that should be executed on mainnet forks and Hardhat).

Reply:
```typescript
    // Execute the proposals which have already been executed on mainnet.
    //
    // The proposals will be executed when running on a local test network,
    // but will not be executed when running on a mainnet fork.
    
    ...

    // Execute the proposals which have not yet been executed on mainnet.
```